<?xml version="1.0"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta name="generator" content=
    "HTML Tidy for NetBSD (vers 1st March 2002), see www.w3.org" />
<title>Day 14 Morning</title>
    <meta name="description" content=
    "Alison Chaiken's home page" />
    <meta name="keywords" content=
    "molecular biology, hybridization, cloning, vector, PCR, DNA,
    fingerprinting, RNA, mRNA, cDNA, primer, polymerase, E. coli,
    operon, promoter, plasmid, gel, electrophoresis, southern blot,
    yokazaki, Eco R1, restriction enzyme, intron, exon, siRNA, gene
    expression, array, 5' UTR, poly-A tail, tRNA, translation, transcription" />
    <meta name="resource-type" content="document" />
    <meta name="distribution" content="global" />
    <meta http-equiv="Content-Type" content=
    "text/html; charset=utf-8" />
    <meta name="GENERATOR" content=
    "GnuEmacs/19.34 [en] (X11; I; NetBSD 1.2G i386) [emacs]" />
    <link rev="made" href=
    "mailto:alchaiken@gmail.com" />
    <link rel="start" href="/" />
    <link rel="icon" href="/fermi-16.png" type=
    "image/png" />
    <link rel="stylesheet" href="/pt_style.css" type="text/css" />
    <!--    $Id: index.html,v 1.25 2005/10/22 20:58:32 alison Exp $  -->
    <link rel="up" href="../" />
    <link rel="next" href="coolstuff.html" />
    <link rel="up" href="/" />
<!--    $Id: day14am.html,v 1.2 2004/07/04 06:05:35 alison Exp $  -->
</head>
<body>
<a href="index.html">Up</a>
<a href="day13pm.html">Previous</a>

<h2><p align="center">Day 14 Morning Lecture Notes</p></h2> <h3><p
align="center"> Steve Williams, Smith College</p></h3><h3><p
align="center">June 19, 2004</p></h3>

<h3>Interpretation of genomic Southern blot results (p. 94)</h3>

<p align=center>
<img src="southerndigest.png">
</p>

<p>The lengths of Southern blot digest fragments give clues about the
location of restriction sites in a particular gene.  There must be at
least one Msp site within RevT, as otherwise the RevT probe would not
hybridize to lower-MW bands of the Msp digest.  The difference between
Hpa and Msp (isoschizomers with the same recognition site) shows that
mouse RevT is heavily methylated.  Methylated genes are expressed less
heavily, which certainly makes sense for RevT.
</p>

<p>Once lambda DNA gets methylated, one strand stays methylated, which
is enough to protect it against restriction enzymes.  The extra
stability of this dsDNA allows enough time for the other strand to be
methylated, so lambda starts to reproduce quickly since restriction
enzymes can't attack it once its DNA is methylated.
</p>

<p>The source of methyl groups is methionine whose dietary source is
folic acid.  DNA methylation can be prevented by a diet low in
methionine.   E. coli can grow in a medium which supplies only
glucose, biotin and salts.
</p>

<h3>DNA sequencing results</h3>

<p>Some EcoR1-cut mouse DNA that was sequenced was not cut at the two
EcoR1 sites that are closest to the RevT gene but was cut at a
different EcoR1 site.  The EcoR1 site close to RevT should be present
downstream in these sequences.  As long as RevT occurs somewhere in
the sequence, the plaque will still be positive.
</p>

<p>Unincorporated fluorescent ddNTP*s run at a particular MW and line
up in sequences.  This is not surprising since the dyes are such large
molecules.  The error rate in PCR copying is only 1-2x10<sup>-5</sup>.
"Proofreading" polymerases will recognize an error and correct it.
Taq is not a proofreading polymerase and has a slightly higher error
rate but also has a higher processivity.
</p>

<h3>Targeted deletions</h3>

[See the handout labeled "Change 4 nt" and "Delete 7 nt"] 

<p>Targeted deletions can be used to study how promoters work and to
change amino acids in a protein.  Targeted deletions or additions work
by intentionally using a slightly incorrect sequence as a PCR primer.
To change the middle of a long gene far from where the primers must
be, cut out the middle of the gene.    Mutate the excised segment,
then paste it back in using restriction enzymes.    
</p>

<p>To see if a particular protein binds to a particular sequence, try
an affinity column.    Or try a gel affinity assay to see if adding
the protein to DNA shifts its band.     The band can be cut from DNA
and analyzed to confirm protein-DNA binding.
</p>

<h3>PCR variants</h3>

<p>Inverse PCR is a technique that uses the known sequence of a gene
to learn sequence of the surrounding material.  The PCR primers are
both derived from the sequence of the surrounding known material.
</p>

<p align=center>
<img src="inversePCR.png">
</p>

<p>"Long" PCR from Applied Biosystems used 2 polymerases in the same
tube to get longer chains.  One is a proofreading enzyme and one is a
more processive enzyme that does not proofread.  The processivity of
Taq polymerase is about 10 bp.  Multiple Taqs will process a given
strand during extension.
</p>



<a href="index.html">Up</a>
<a href="day13pm.html">Previous</a>

</body>
</html>
