<?xml version="1.0"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta name="generator" content=
    "HTML Tidy for NetBSD (vers 1st March 2002), see www.w3.org" />
<title>Day 12 Afternoon</title>
    <meta name="description" content=
    "Alison Chaiken's home page" />
    <meta name="keywords" content=
    "molecular biology, hybridization, cloning, vector, PCR, DNA,
    fingerprinting, RNA, mRNA, cDNA, primer, polymerase, E. coli,
    operon, promoter, plasmid, gel, electrophoresis, southern blot,
    yokazaki, Eco R1, restriction enzyme, intron, exon, siRNA, gene
    expression, array, 5' UTR, poly-A tail, tRNA, translation, transcription" />
    <meta name="resource-type" content="document" />
    <meta name="distribution" content="global" />
    <meta http-equiv="Content-Type" content=
    "text/html; charset=utf-8" />
    <meta name="GENERATOR" content=
    "GnuEmacs/19.34 [en] (X11; I; NetBSD 1.2G i386) [emacs]" />
    <link rev="made" href=
    "mailto:alchaiken@gmail.com" />
    <link rel="start" href="/" />
    <link rel="icon" href="/fermi-16.png" type=
    "image/png" />
    <link rel="stylesheet" href="/pt_style.css" type="text/css" />
    <!--    $Id: index.html,v 1.25 2005/10/22 20:58:32 alison Exp $  -->
    <link rel="up" href="../" />
    <link rel="next" href="coolstuff.html" />
    <link rel="up" href="/" />
<!--    $Id: day12pm.html,v 1.2 2004/07/04 06:05:35 alison Exp $  -->
</head>
<body>
<a href="index.html">Up</a>
<a href="day12am.html">Previous</a>
<a href="day12eve.html">Next</a>

<h2><p align="center">Day 12 Afternoon Lecture Notes</p></h2> <h3><p
align="center"> Steve Williams, Smith College</p></h3><h3><p
align="center">June 17, 2004</p></h3>

<h3>Cloning a protein whose sequence is unknown</h3>

<p>How do we clone a protein when we don't know the sequence and the
antibody response to it in rabbit fails?  Commercial vendors do amino
acid sequencing of the ends of proteins.  Typically they can get 12
amino acids at the amino terminus of the protein and 12 internally
from a tryptic digest.    A primer can be designed via reverse-lookup
in the genetic codon table.
</p>

<p>Consider for example a protein amino terminus with the sequence
lys-phe-pro-tyr-his-ile-met.  Lys can be coded for with 2 different
sequences, phe with two, pro with 4, tyr with 2, his with 2, ile with
3 and met with 1 for a total of 192 distinct possible starting
sequences.  If we knew the correct sequence, we could make an oligo
probe.  For about $100, vendors will prepare all 192 in a
"mixed-synthesis oligo" product that can be used to screen a cDNA
library.  (You wouldn't want to use mixed-synthesis oligos to screen a
genomic library as there would be too many false positives.)  Of the
192 oligos, more than one may well bind.
</p>

<p>Now take 7 amino acids from the tryptic digest and hybridized with
mixed-synthesis oligos again on another lift from the same plate.
Anything that hybridizes with both mixed-synthesis oligos is likely to
be the protein of interest.  Of course the positive clones may only
contain part of the gene.
</p>

<p>There is a protein BLAST similar to the program for genomes.   Gene
expression products should be checked against this database.  It's
very difficult to sequence a full-length protein even with mass-spec
or 2D gels.  Most sequences in the protein database come from the
genome.   There's also a BlASTX program that find proteins which match
a section of genomic DNA.
</p>

<h3>Screening a cDNA library</h3>

<ul>

<li>Use a DNA probe, perhaps from related species.</li>

<li>Antibody from a rabbit;</li>

<li>Mixed oligos.</li>

</ul>

<p>Note that only the DNA probe works with a genomic library, which is
why cDNA libraries are often made first.
</p>

<p>Two common methods of studying gene expression and are
reverse-transcriptase PCR and quantitative PCR.  Another possibility
is primer extension.
</p>

<p>The LINE1 repeat is a "molecular parasite" that reproduces itself.
Only 2% of DNA encodes proteins.   RNA splicing eliminates intron
loops to turn pre-mRNA into mRNA.   "Alternative splicing" can give
rise to different mRNAs.   Commonly alternative splicing hooks
different modular domains together in different configurations.
</p>

<p>In humans a gene is about 1300 nt on average with 9 exons.  2/3 of
genes have "splice variants," a higher fraction than in other
organisms.  Humans and other mammals have more protein diversity from
the same domains than simpler organisms.  40% of human genes have an
unknown function.  A large fraction of all identified genes are the
housekeeping ones.  The genes unique to humans primarily are for
immunology, neural development, blood clotting, transcription factors
and regulatory proteins.   The human proteome has perhaps
10<sup>5</sup> proteins.
</p>

<p>
</p>

<a href="index.html">Up</a>
<a href="day12am.html">Previous</a>
<a href="day12eve.html">Next</a>

</body>
</html>
