<?xml version="1.0"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta name="generator" content=
    "HTML Tidy for NetBSD (vers 1st March 2002), see www.w3.org" />
<title>Day 2 Afternoon</title>
    <meta name="description" content=
    "Alison Chaiken's home page" />
    <meta name="keywords" content=
    "molecular biology, hybridization, cloning, vector, PCR, DNA,
    fingerprinting, RNA, mRNA, cDNA, primer, polymerase, E. coli,
    operon, promoter, plasmid, gel, electrophoresis, southern blot,
    yokazaki, Eco R1, restriction enzyme, intron, exon, siRNA, gene
    expression, array, 5' UTR, poly-A tail, tRNA, translation, transcription" />
    <meta name="resource-type" content="document" />
    <meta name="distribution" content="global" />
    <meta http-equiv="Content-Type" content=
    "text/html; charset=utf-8" />
    <meta name="GENERATOR" content=
    "GnuEmacs/19.34 [en] (X11; I; NetBSD 1.2G i386) [emacs]" />
    <link rev="made" href=
    "mailto:alchaiken@gmail.com" />
    <link rel="start" href="/" />
    <link rel="icon" href="/fermi-16.png" type=
    "image/png" />
    <link rel="stylesheet" href="/pt_style.css" type="text/css" />
    <!--    $Id: index.html,v 1.25 2005/10/22 20:58:32 alison Exp $  -->
    <link rel="up" href="../" />
    <link rel="next" href="coolstuff.html" />
    <link rel="up" href="/" />
<!--    $Id: day2pm.html,v 1.2 2004/07/04 06:05:35 alison Exp $  -->
</head>
<body>

<a href="index.html">Up</a>
<a href="day2am.html">Previous</a>
<a href="day2eve.html">Next</a>

<h2><p align="center">Day 2 Afternoon Lecture Notes</p></h2> <h3><p
align="center"> Steve Williams, Smith College</p></h3><h3><p
align="center">June 7, 2004</p></h3>

<p>The RevT gene is found in a repeating sequence called LIN1 (for
long-interspersed sequence 1) which is 7 kb long.  LIN1 is a reverse
transposon and is scattered around all the chromosomes.  Transposons
splice themselves in and out of genes and change position.
Retrotransposons are so-called because they move via an RNA
intermediate.  Retroviruses may have evolved from retrotransposons.
</p>

<p>LIN1 accounts for >10% of the mouse genome.  The 7 kbp sequence has
two EcoR1 sites.  The actual RevT coding sequence of 1.4 kbp lies
between these EcoR1 sites.  The huge number of RevT repeats in the
genome gives a distinct band in an EcoR1 digest at 1.4 kbp.  Stained
gels can be misleading because higher MW fragments bind more ethidium
bromate dye than smaller fragments.  Not all retrotransposons contain
a sequence that will code for the reverse transcriptase protein; some
are non-functional "pseudogenes."  Reverse transcriptase makes DNA
from RNA.
</p>

<p>SINEs are "short interspersed repeats" or "short tandem repeats."
The famous ALU repeat is 300 base pairs long.   Repeated sequences
make up about 50% of mammalian genomes.
</p>

<p>Wild-type lambda phage has 5 EcoR1 sites.  The genetically
engineered cloning vector Lambda Zap Express has one precisely
positioned EcoR1 site separating the longer "left arm" (28.3 kbp) and
shorter "right arm" (10.5 kbp).  In experiment 1 we will insert the
mouse RevT gene between the two arms of Lambda Zap Express.  DNA
ligase will will form new phosphodiester bonds between the 3' and 5'
ends of the backbone for a total of 4 ligations.  The mouse DNA is
then integral with the lambda DNA.
</p>

<p>During such a process, misligations (mouse-mouse, right-left,
left-left, circular mouse, etc.) will occur.  Only left-mouse-right is
viable along with left-right.  Other possibilities are missing some of
the genes lambda needs to reproduce.  How can we prevent the
religation of the two lambda arms without the mouse insert?
</p>

<p>We use calf intestinal alkaline phosphatase (CIAP).  This enzyme
removes phosphates from the 5' ends of the vector.  Now the left and
right arms, like phosphates, can't religate to each other.  The mouse
insert isn't treated with CIAP so it retains the 5' phosphates and the
ability to ligate to the left and right arms.  Only insert-5' to
vector-3' will work as the vector-5' has no phosphate.  With 2 out of
the 4 backbones bonded along with the bases, the molecule will be
reasonably stable.  When the ligated molecule is put into E. coli, a
cellular enzyme will repair it.
</p>

<p>Nothing mentioned so far will prevent the insertion of more than
one mouse RevT gene.  This misligation is prevented by using few
insert molecules in the reaction.  To get 1:1 insert:vectors, set (ng
vector/length vector) = (ng insert/length insert).  Following p. 36 of
the lab protocols, (200 ng insert/4000 bp)/(1000 ng
vector/39x10<sup>3</sup> bp) = 2, the maximum amount advisable.  Here
the high 2:1 ratio accounts for loss in dialysis and other procedures.
</p>

<p>How is Lambda Zap Express modified from wild-type phage?  It has
only one EcoR1 site and its length is reduced from 48.5 kbp to 39 kbp
by removing untranscribed DNA.  Due to limited space in the head of
the phage, the maximum length of the lambda genome is 51 kbp.
Therefore LZE can accommodate inserts up to 12 kbp long as opposed to
only 2.5 kbp for wild-type phage.  The molecular motor which packages
DNA into the head doesn't recognize a chromosome shorter than 39 kbp,
so that's the effective lower bound.
</p>

<p>The LZE product is DNA which is already cut and dephosphorylated
(i.e. terminal phosphate groups are removed).  Nonetheless there is
still a small amount of viable left-right ligation product without an
insert.
</p>

<p>Why do we have to heat the lambda MW markers before running them on
the gel?  In wild-type lambda, the 5' ends have a 12-nucleotide (nt)
overhang.  These "COS" ends can ligate and form a ring.  Heating the
lambda marker unbinds these ends and linearizes the DNA.  65C is hot
enough to unbind the 12 hydrogen bonds that hold the ends together.
Various restriction enzyme digests of lambda (Lambda BSTEII, Lambda
Hind for example) are useful as they give a number of predictable
bands.  The fragments would bind together to form higher MW products
if the markers were not heated.  The enzyme that packs phage DNA into
the head also linearizes the circular form.
</p>

<p>Viable lambda bacteriophage can easily be assembled in a test tube:
mix proteins, stir and wait 2 hours.  Stratagene (San Diego) and
Promega both sell a "packaging extract" that include head proteins,
tail proteins and assembly proteins.  Just add DNA and you have all
the ingredients to make phage.  The DNA must have 39 kbp <= length <=
51 kbp; it must have both the left and right arms; and it must have
the COS ends.
</p>

<p>Other methods besides phage and plasmids can insert DNA into
E. coli, including electroporation and "gene guns," but phage
infection is by far the most efficient in term of cells transformed
per unit starting DNA.   Phage form clear "plaques" on the "bacterial
lawn" of a Petri dish.   Each plaque may have a different insert from
the mouse genome.   Special E. coli strains with no restriction
sites in their genome are used for transformation.   A normal Petri
dish can accommodate about 10<sup>4</sup> inserts.
</p>

<p>Since the RevT gene is about 10% of the mouse genome, a plate with
a hundred plaques will almost always have a RevT plaque.  A
single-copy gene will require a much larger plate in order to find a
plaque.  The reason why plaques self-limit in size is not known.  The
phage must be low enough in concentration that the plaque boundaries
can be visualized.
</p>

<a href="index.html">Up</a>
<a href="day2am.html">Previous</a>
<a href="day2eve.html">Next</a>

</body>
</html>
