<?xml version="1.0"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta name="generator" content=
    "HTML Tidy for NetBSD (vers 1st March 2002), see www.w3.org" />
<title>Day 2 Evening</title>
    <meta name="description" content=
    "Alison Chaiken's home page" />
    <meta name="keywords" content=
    "molecular biology, hybridization, cloning, vector, PCR, DNA,
    fingerprinting, RNA, mRNA, cDNA, primer, polymerase, E. coli,
    operon, promoter, plasmid, gel, electrophoresis, southern blot,
    yokazaki, Eco R1, restriction enzyme, intron, exon, siRNA, gene
    expression, array, 5' UTR, poly-A tail, tRNA, translation, transcription" />
    <meta name="resource-type" content="document" />
    <meta name="distribution" content="global" />
    <meta http-equiv="Content-Type" content=
    "text/html; charset=utf-8" />
    <meta name="GENERATOR" content=
    "GnuEmacs/19.34 [en] (X11; I; NetBSD 1.2G i386) [emacs]" />
    <link rev="made" href=
    "mailto:alchaiken@gmail.com" />
    <link rel="start" href="/" />
    <link rel="icon" href="/fermi-16.png" type=
    "image/png" />
    <link rel="stylesheet" href="/pt_style.css" type="text/css" />
    <!--    $Id: index.html,v 1.25 2005/10/22 20:58:32 alison Exp $  -->
    <link rel="up" href="../" />
    <link rel="next" href="coolstuff.html" />
    <link rel="up" href="/" />
<!--    $Id: day2eve.html,v 1.2 2004/07/04 06:05:35 alison Exp $  -->
</head>
<body>
<a href="index.html">Up</a>
<a href="day2pm.html">Previous</a>
<a href="day3am.html">Next</a>

<h2><p align="center">Day 2 Evening Lecture Notes</p></h2> <h3><p
align="center">Barton Slatko, New England Biolabs</p></h3><h3><p
align="center">June 7, 2004</p></h3>

<h3>Modification by Restriction Enzymes</h3>

<p>When phage are killed by bacteria they are trying to infect, they
were historically said to be "restricted."  "Plaques" are "holes in
the bacterial lawn" of a petri dish and are comprised of phages that
are not killed.  Survivor phage can be modified and used to infect
another lawn in an iterative process until all phage survive.  The
phage were historically said to be "modified."
</p>

<p>If the modified survivor phage are now exposed to a different
strain of E. coli, they will once again be killed.  Once again
modified phage can be isolated.  Bacteria have "restriction enzymes"
that kill attacking phages.  Restriction endonucleases cut phage DNA
internally.  The phages that survive are methylated -- that is, a
methyl group protects DNA from cutting.  In this example the second
strain of E. coli has a different restriction enzyme that cuts at
another still unmethylated site.
</p>

<p>Bacteria have these cleavage enzymes for self-protection.  Where
does the methylation defense come in?  A cell doesn't want to cut its
own DNA so it has evolved a "cognate methylase" to protect its own
DNA.
</p>

<p>Some bacteria have evolved as many as 12 restriction enzymes.  The
proliferation is a sign of the arms race between bacteria and viruses.
Usually methylation is slower than restriction but occasionally phage
progeny are still produced.  Most progeny are killed but some survive
since the methylated ssDNA template created during replication will
have a greater tendency to get a methylated second strand, which in
turn will favor another methylated strand during the next replication.
(This propagating methylation appears to be a unique example of
Lamarckian inheritance of an acquired characteristic.)
</p>

<p>EcoR1 comes from E. coli, strain R.  Other enzymes come from
different bacteria.  A few hundred restriction enzymes have been
identified that fall into four classes.  The type II are most commonly
used.  Type II enzymes can be monomers (like methylase) or dimer
(endonucleases).  Many recognize like EcoR1 recognize dyad (mirror)
sequences.  Dimers sit on both strands of DNA simultaneously.
Cleavage enzymes stress and distort bonds until they break.  The
recognition mechanism that enzymes use is heavily studied.  A methyl
group on the second nt of the EcoR1 site (the first A of GAATTC)
prevents EcoR1 from cutting it.  Each restriction enzyme has its own
methylase.  Methyl transferase moves methyl groups from S-adenosyl
methionine (SAM) to DNA.  SAM is often used in buffer solutions.
</p>

<p>In general restriction enzymes can leave overhangs at the 5' or 3'
end, or they can be "blunt cutters."    Clearly enzymes that leave
"blunt ends" have less specificity in religation than those that
produce matching overhangs.   With EcoR1, two overhangs that ligate
together can be cut again with the same enzyme.   This is not true
with blunt cutters, where a blunt of the recognition site
</p>

<p align="center">
5' GGGCCC 3'<br>
3' CCCGGG 5'
</p>
                          
will produce 

<p align="center">
5' GGG CCC 3'<br>
3' CCC GGG 5'
</p>

<p>where the two ends can now bind to another sequence that will not
reproduce the GGGCCC recognition site.  
</p>

<p>Example: HincII cuts </p>

<p align="center">
5' GTPyPuAC 3'<br>
3' CAPuPyTG 5'
</p>

<p>
where "Py" is any pyrimidine and "Pu" any purine.  This
non-specificity is called "enzyme wobble."
</p>

<p>Note that an enzyme that cuts at the center of a recognition site
is a blunt cutter while one that cuts nearer an end produces an
overhang.  Less-specific blunt cutters are harder to use because of
more possibilities for undesirable ligation products but convenient
recognition sites for enzymes that produce overhangs are not always
available in the region of interest.  Enzymes that cut near the
beginning of a recognition sequence (like EcoR1) generate a 3'
overhang while those that cut near the end generate a 5' overhang.
</p>

<p>The length of recognition sites varies.  To cut DNA into many small
fragments use an enzyme with a short recognition sequence, e.g. a
"4-cutter."  To cut DNA into fewer, longer fragments, use an enzyme
with a longer recognition sequence like an "8-cutter."
</p>

<p>For a random DNA sequence, the probability of finding the EcoR1
recognition site is 0.25<sup>4</sup> = 1/4096.  However, genomes are
not truly even in nucleotide distribution.  Thermophile organisms have
more of the strongly bonded GC nucleotides while many organisms have
more than 50% AT.
</p>

<p>Two enzymes that have the same restriction sequence are called
"isoschizomers."  They may still cut at different places and generate
different resulting overhangs.  For example HhaI and HinPI both cut
GCGC, but one generates a 3' overhang and one a 5' overhang.  The New
England Biolabs catalog has a chart explaining which enzymes produce
compatible overhangs.
</p>

<p>A major difficulty in developing a cloning strategy is that much of
mammalian and plant DNA is methylated.  Different restriction enzymes
are blocked by different methylation sites even if they have the
recognition sequence.  If the sequence of the gene to be cloned is not
known, the scientist is advised simply to try a bunch of different
restriction enzymes.
</p>

<p>Except right after replication, methylation is always symmetric on
both strands.  This is why methylated phages tend to have methylated
progeny.   
</p>

<h3>Restriction Enzyme Strategy -- A Summary</h3><p>

<p>In choosing an enzyme, be aware of:
</p>

<ul>

<li>GC/AT content;<p>

<li>methylation status of insert and vector;<p>

<li>enzyme compatibility including overhangs and isoschizomers;<p>

<li>reaction conditions, including buffer;<p>

<li>benefits and hazards of glycerol, which allows storage at lower
temperatures but causes reduced enzyme specificity.

</ul>

<a href="index.html">Up</a>
<a href="day2pm.html">Previous</a>
<a href="day3am.html">Next</a>


</body>
</html>
