<?xml version="1.0"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta name="generator" content=
    "HTML Tidy for NetBSD (vers 1st March 2002), see www.w3.org" />
<title>Day 3 Afternoon</title>
    <meta name="description" content=
    "Alison Chaiken's home page" />
    <meta name="keywords" content=
    "molecular biology, hybridization, cloning, vector, PCR, DNA,
    fingerprinting, RNA, mRNA, cDNA, primer, polymerase, E. coli,
    operon, promoter, plasmid, gel, electrophoresis, southern blot,
    yokazaki, Eco R1, restriction enzyme, intron, exon, siRNA, gene
    expression, array, 5' UTR, poly-A tail, tRNA, translation, transcription" />
    <meta name="resource-type" content="document" />
    <meta name="distribution" content="global" />
    <meta http-equiv="Content-Type" content=
    "text/html; charset=utf-8" />
    <meta name="GENERATOR" content=
    "GnuEmacs/19.34 [en] (X11; I; NetBSD 1.2G i386) [emacs]" />
    <link rev="made" href=
    "mailto:alchaiken@gmail.com" />
    <link rel="start" href="/" />
    <link rel="icon" href="/fermi-16.png" type=
    "image/png" />
    <link rel="stylesheet" href="/pt_style.css" type="text/css" />
    <!--    $Id: index.html,v 1.25 2005/10/22 20:58:32 alison Exp $  -->
    <link rel="up" href="../" />
    <link rel="next" href="coolstuff.html" />
    <link rel="up" href="/" />
<!--    $Id: day3pm.html,v 1.2 2004/07/04 06:05:35 alison Exp $  -->
</head>
<body>
<a href="index.html">Up</a>
<a href="day3am.html">Previous</a>
<a href="day4am.html">Next</a>

<h2><p align="center">Day 3 Afternoon Lecture Notes</p></h2> <h3><p
align="center"> Steve Williams, Smith College</p></h3><h3><p
align="center">June 8, 2004</p></h3>

<p><h3>How to screen a library for a gene of interest</h3></p>

<p>The standard way of screening is to use a DNA probe.   Most
commonly the probe is a cDNA clone.   Why even bother to look at a
genomic library if a cDNA is already available?    Often the reason is
to find out about non-coding regulatory sections of the gene:
promoters, operators, etc.   A cDNA library is easier to screen and
prepare but a genomic library contains much more information.
</p>

<p>A second type of DNA probe is a heterologous (het-er-AHL-ug-us)
probe such as a gene from a closely related species.  Another example
would be to use DNA from a different gene from the same species and
family.  For example, adult hemoglobin DNA could be used to screen for
the fetal hemoglobin gene.  The RevT family may have as many as
10<sup>5</sup> members.  We will use one RevT clone to find other
members of the same gene family.  In general about 70% of a sequence
has to match in order for probing to work but only one portion of the
probe has to match closely.
</p>

<ol>

<li>Select DNA probe and label.</li>

<p>Radioactive labels used to be most common but safety concerns and
disposal regulations are making these less common.  The popular
isotopes were <sup>32</sup>P, <sup>35</sup>S and <sup>33</sup>P.  More
common now are fluorescent probes like fluorescein (floor-eh-scene),
biotin and digoxigenin.   Fluorescent probes are must faster than
radioactive probes which can at times be more sensitive.
</p>

<li>Plate out library and perform plaque lifts.</li>

<p>How many plaques do we need to test in order to have a 99% chance
of getting one with a given single-copy gene if there are 1 million
fragments?  Typical laboratory practice is 7X coverage, that is 7
million clones screened for 1 million distinct fragments.   A typical
plate (6" x 6") holds about 50,000 plaques.
</p>

<p>Our mouse genomic library cut by EcoR1 in experiment 1 has about 1
million fragments.  RevT is much easier to screen for due to the many
copies in the genome.  >= 100 plaques should guarantee success.
</p>

<p>In order to perform the plaque lift, place a nylon membrane on the
surface of the plate.   The membrane sucks up liquid including phage
from each plaque.    Peel the membrane off, taking about 10% of the
phage from each plaque, about 10<sup>5</sup> in number. 
</p>

<li>Treat plaque lifts with NaOH.</li>

<p>The phage are lysed and DNA is denatured by NaOH.  (In other words,
DNA goes from double-stranded to single-stranded, but the individual
strands aren't broken.)   ssDNA is now stuck to the membrane in the
position of all the original plaques.
</p>

<li>Neutralize the membrane to pH 7.</li>

<li>Irreversibly bind the DNA to the membrane.</li>

<p>The binding can be done with UV crosslinking in a dedicated chamber
that delivers a controlled low does.   Alternatively the membrane can
be baked 1-2 hours at 80&#176;C.
</p>

<li>Prehybridization.</li>

<p>Expose the membrane to salmon sperm ssDNA fragments 100-200
nucleotides (nt) in length.   This passivation step prevents probe DNA
from binding non-specifically to the membrane.   Salmon sperm DNA is
used because of its low price.   Yeast RNA is an alternative.
</p>

<li>Hybridization.</li>

<p>Denature the probe DNA and add it to the bag containing the
membrane and the prehybridization solution.
</p>

<li>Detect the bound probes using a method appropriate to the
labeling method.</li>

<li>Compare positive plaques to plate.

<p>Use a pipette or needle to remove the corresponding phage and
transfer to a tube of buffer solution.  If a screen shows that 3 or 4
plaques are positive, sequence all the DNA to see which one has the
gene of interest.
</p>

<li>DNA that is known to be complementary to the probe can be spotted
onto the membrane as a positive control to make sure that the
hybridization and detection steps work properly.</li>

</ol>


<h3>Introduction to the Polymerase Chain Reaction (PCR)</h3>

<p>The three major discoveries in molecular biology are the discovery
of the structure of DNA in the 1950's, the development of early
cloning methodologies in the 1970's, and the invention of PCR in
1986.   Cloning recombinant DNA has 2 basic objectives: to identify a
gene of interest in a complex mixture; and to "amplify" the gene and
make many copies.
</p>

<p>Consider the pgk2 single-copy gene of mouse.  pgk2 is 3 kbp long
out of 3-4x10<sup>9</sup> in the total mouse genome.  With PCR all we need
to know to amplify the gene is the beginning and the end of the pgk2
sequence so that we can make oligonucleotide primers.   Primers can be
purchased for as little as $20 for a 20-30 bp sequence.   As small a
primer as 15 bp may work.   Turnaround time for primer orders can be
as short as 48 hours.
</p>

<p align="center">
<img src="pcrprimers.png" height=193 width=678>
</p>

<h3>Rules for DNA Synthesis</h3>

<ol>

<li>always proceeds 5' --> 3';</li>

<li>always requires a primer that has a free 3' end;</li>

<li>the coding strand is always the same; only one of the strands can
ever be transcribed in a given part of the sequence.</li>

</ol>

<p>RNA synthesis can proceed without a primer but DNA synthesis
cannot.  Note that the primers determine uniquely what sequence is
amplified.   
</p>

<h3>PCR Thermal Cycling Reaction</h3>

<ol>

<li>Heat to 94&#176;C for one minute to <b>denature</b> the dsDNA;</li>

<li>Cool for one minute to a temperature where primers will hybridize,
usually 45&#176;C <= T <= 70&#176;C depending on primer length and
composition.   This is the <b>annealing</b> step.</li>

<li>Heat to 72&#176;C for Taq polymerase to synthesize new DNA.   This
is the <b>extension</b> step.</li>

</ol>

<p>Taq polymerase comes from the thermophilic bacterium Thermus
aquaticus from Yellowstone.
</p>

Ingredients needed for PCR:

<ul>

<li>template DNA;</li>
<li>Taq polymerase;</li>
<li>Two kinds of primers (forward and reverse);</li>
<li>the nucleotide triphosphates;</li>
<li>MgCl<sub>2</sub>;</li>
<li>an appropriate buffer.</li>

</ul>

<p>Note that the two primers are <b>not</b> complementary sequences.
They are rather the different sequences of the two ends of the gene.
Since DNA amplification cannot occur without a primer, impurity DNA in
the solution does not get amplified.   In the DNA replication process
during normal cell division, an RNA primer is synthesized.
</p>   

<p>Multiplex PCR means amplifying many sequences at once.  Using
standard PCR a sequence up to a few thousand bp in length can be
amplified.  Special tricks can extend the limit to 20 kbp.  PCR
doubling tends to stop when the likelihood of two single strands
finding each other is greater than their likelihood of finding
primers.  Usually this saturation occurs at about 30-40 cycles after
1.5-2 hours.
</p>

<a href="index.html">Up</a>
<a href="day3am.html">Previous</a>
<a href="day4am.html">Next</a>


</body>
</html>
