<?xml version="1.0"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta name="generator" content=
    "HTML Tidy for NetBSD (vers 1st March 2002), see www.w3.org" />
<title>Day 7 Morning</title>
    <meta name="description" content=
    "Alison Chaiken's home page" />
    <meta name="keywords" content=
    "molecular biology, hybridization, cloning, vector, PCR, DNA,
    fingerprinting, RNA, mRNA, cDNA, primer, polymerase, E. coli,
    operon, promoter, plasmid, gel, electrophoresis, southern blot,
    yokazaki, Eco R1, restriction enzyme, intron, exon, siRNA, gene
    expression, array, 5' UTR, poly-A tail, tRNA, translation, transcription" />
    <meta name="resource-type" content="document" />
    <meta name="distribution" content="global" />
    <meta http-equiv="Content-Type" content=
    "text/html; charset=utf-8" />
    <meta name="GENERATOR" content=
    "GnuEmacs/19.34 [en] (X11; I; NetBSD 1.2G i386) [emacs]" />
    <link rev="made" href=
    "mailto:alchaiken@gmail.com" />
    <link rel="start" href="/" />
    <link rel="icon" href="/fermi-16.png" type=
    "image/png" />
    <link rel="stylesheet" href="/pt_style.css" type="text/css" />
    <!--    $Id: index.html,v 1.25 2005/10/22 20:58:32 alison Exp $  -->
    <link rel="up" href="../" />
    <link rel="next" href="coolstuff.html" />
    <link rel="up" href="/" />
<!--    $Id: day7am.html,v 1.2 2004/07/04 06:05:35 alison Exp $  -->
</head>
<body>
<a href="index.html">Up</a>
<a href="day6pm.html">Previous</a>
<a href="day9am.html">Next</a>

<h2><p align="center">Day 7 Morning Lecture Notes</p></h2> <h3><p
align="center"> Steve Williams, Smith College</p></h3><h3><p
align="center">June 12, 2004</p></h3>

<p>RNA and DNA form stronger bonds to each other than DNA-DNA bonding.
Primers used in eukaryotic DNA replication are also RNAs.   Two
strands of RNA can form a double helix.   Some viruses have
double-stranded RNA genomes.
</p>

<p>How much insert and vector do we want to use in a ligation
reaction?  Ideally (ng insert/MW insert)/(ng vector/MW vector) = 7.5.
We used 25 ng of a 3850 bp vector, so for a 1600 bp RevT insert, we
need about 80 ng of insert to get the optimal ratio.  We are able to
use a high insert/vector ratio since the inserts cannot bind to each
other due to their having an EcoR1 overhang on one end and an Xho
overhang on the other.
</p>

<h3>Sources of PCR Contamination</h3>

<p>A genomics lab that works with mouse is liable to be contaminated
with mouse DNA, RNA, cDNA, genes cloned in lambda, genes cloned in
plasmids, and previous PCR products.  The latter are the biggest
problem.  Because DNA is so stable, it's on all lab surfaces at all
times.  A central facility lab that works with many species has fewer
concerns about contamination.
</p>

<p>Every time a microfuge tube is opened, a contaminating aerosol is
generated.  Puling aerosols into unfiltered tips is also a problem.
</p>

<p>Viruses with lipid polysaccharide coats on their surface are
destroyed by air.  Those with protein coats like lambda are
air-stable.
</p>

<h3>Common steps to help reduce contamination in PCR reactions<br> (in
order of increasing difficulty and expense; see also p. 123-125 of
handouts)</h3>

<ol>

<li>Always wear gloves and change them frequently.</li>

<li>Use filter-barrier pipet tips.</li>

<li>Use a dedicated set of pipets for the PCR setup.  Never use pipets
that were used with PCR products to set up a new PCR reaction.</li>

<li>Have a dedicated room or bench for PCR prep and handle PCR
products elsewhere.</li>

<li>Clean lab benches, equipment and pipets frequently with a 5%
bleach solution from a spray bottle.  Bleach causes DNA to cross-link.
</li>

<li>Aliquot PCR reagents to each individual investigator.  That way
contamination sources can be identified and contained.
</li>

<li>Add template to PCR reactions last.  Mix all other reagents first
and put the containers away.  Then bring out the template and add it
to the mix.  Or at least store the template at the other end of the
bench from the reagents.
</li>

<li>Use uracil n-glycosylase (UNG).  How does this work?
</li>

<p> UNG in cells removes U's from DNA and (in the presence of dTTP)
replaces them with T's.  A U-containing gene cloned into a vector will
have them replaced by T's.  In vitro UNG only removes U's, fragmenting
the DNA.
</li>

<p>Suppose we run the initial reverse transcription step of RT-PCR
with dTTP but use dUTP for all subsequent PCR amplifications.  Any
contaminating product from a previous PCR reaction is therefore
damaged by UNG.  The real cDNA template from the reverse transcription
reaction will not be affected by the UNG.  This UNG trick was
developed by Applied Biosystems.  However any contamination with
genomic DNA will still contain T's and can still be amplified.
</li>

<li>Use a tissue-culture hood to mix reagents.  These hoods blow
sterile air down out of the hood.  They also have UV lights for
cross-linking that must be turned off when template is handled (!).
Similar bench-top PCR hoods are available.   Usually these hoods are
overkill.
</li>

<li>Special RNAse sprays and wipes are available, but once again these
are usually overkill.</li>

<p>

</ol> 

<a href="index.html">Up</a>
<a href="day6pm.html">Previous</a>
<a href="day9am.html">Next</a>

</body>
</html>
